iceoryx_posh/include/iceoryx_posh/popo/user_trigger.hpp

```diff
@@ -30,7 +30,7 @@ namespace popo
 class UserTrigger
 {
   public:
-    UserTrigger() noexcept = default;
+    UserTrigger() noexcept;
```

Comment:
Changed because of #494?

Reply:
elfenpiff had also issues compiling this branch on Windows while we were looking why the Windows CI build was broken. But it's a nice side effect that it also fixes that bug. Shall I rebase and change the issue number for that commit?

Reply:
Yes, please :smiley_cat: I suppose #494 can be closed then.

Reply:
#494 maybe shows only the first problem with '() noexcept = default;'-constructor. If you want to close 494, you will need to search in the hole code for '() noexcept = default;'. It was several times in the codebase and I´m not sure if it was remove completely on the master in the meantime.

Reply:
@sculpordwarf Ok, thanks.

Reply:
@sculpordwarf @mossmaurice moved the discussion to #494